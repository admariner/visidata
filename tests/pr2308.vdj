#!vd -p
{"sheet": null, "col": null, "row": null, "longname": "open-file", "input": "tests/small.json", "keystrokes": "o", "comment": null}
{"sheet": "small", "col": "id", "row": "", "longname": "key-col", "input": "", "keystrokes": "!", "comment": "toggle current column as a key column"}
{"sheet": "small", "col": "A", "row": "", "longname": "aggregate-col", "input": "keymax", "keystrokes": "+", "comment": "Add aggregator to current column"}
{"sheet": "small", "col": "", "row": "", "longname": "freq-summary", "input": "", "keystrokes": "zShift+F", "comment": "open one-line summary for all rows and selected rows"}
