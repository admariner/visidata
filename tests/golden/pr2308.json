[
{"Total": "Total", "count": 2, "A_keymax": [2.0]},
{"Total": "Selected", "count": 0}
]
